backend/back-core/src/utils/networks.ts

```
@@ -0,0 +1,15 @@
+
+
+  static isSolana(network: bigint): boolean {
+    switch (network) {
+      case SOLANA_CHAIN_ID:
+      case SOLANA_DEV_CHAIN_ID:
+        return true;
+      default:
+        return false;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)